mantidimaging/gui/ui/recon_window.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1495</width>
```

Comment:
Would it be worth defining the height and weight on a more permanent section as it seems we keep setting new values for this probably by accident in the designer.

Reply:
We can enforce a height/width in the code but the .ui files will keep changing anyway in diffs. 

Not sure whether that will end up being more confusing when someone new comes in and changes the UI file and then doesn't see it actually change